thoth/adviser/cli.py

```diff
@@ -383,7 +383,7 @@ def advise(
     )
 
     parameters["project"] = project.to_dict()
-
+    parameters["runtime_environment"] = parameters["project"]["runtime_environment"]
```

Comment:
The runtime_environment being dict is the requirement. we can also use the json.loads to convert the parameters runtime_environment as well.

Reply:
@pacospace do we use rungime environment from `parameters["project"]["runtime_environment"]` when syncing? 

This issue might be also related to https://github.com/thoth-station/user-api/pull/639

Reply:
Yes correct in sync logic for all components (Adviser, Dependency Monkey and Amun) runtime environment needs to be a `dict` because we use this function https://github.com/thoth-station/storages/blob/ec4ab0b0bff070e415c9f6a4dab50f4c8f1568ff/thoth/storages/graph/postgres.py#L3887

Reply:
> @pacospace do we use rungime environment from `parameters["project"]["runtime_environment"]` when syncing?
> 
> This issue might be also related to [thoth-station/user-api#639](https://github.com/thoth-station/user-api/pull/639)

No for runtime_environment we use `document["result"]["parameters"]`

Reply:
https://github.com/thoth-station/storages/blob/ec4ab0b0bff070e415c9f6a4dab50f4c8f1568ff/thoth/storages/graph/postgres.py#L5079

Reply:
I'm wondering about the other parameters - `requirements` and `requirements_lock`. In the syncing logic there is used `document["result"]["input"]` which is no longer existing. Maybe we should move this change to thoth-storages directly and use `parametrers["project"]` to access runtime environment, requirements and requirements lock for adviser, provenance-checker and dependency monkey during syncs to be consistent. Looking into sources of thoth-adviser, provenance output will need to be adjusted to match the other two.